### PR TITLE
Add release notes - 0.10.4 release

### PR DESCRIPTION
**Pull request by dos65**

### CODE REVIEW

File: website/blog/2021-05-31-tungsten.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    <td>Merged PRs</td>
+    <td align="center">11</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">4</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
```
Comment (tgodzik) [inline website/blog/2021-05-31-tungsten.md:26]: I usually check the last milestone, add any issues that we closed after the last release and use the number there, so for example here: 
https://github.com/scalameta/metals/milestone/39

```suggestion
    <td align="center">3</td>
```

File: website/blog/2021-05-31-tungsten.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
```
Comment (tgodzik) [inline website/blog/2021-05-31-tungsten.md:30]: I would say the 2.12.14 is a new feature? That's what I want to believe anyway :sweat_smile: 

```suggestion
    <td align="center">1</td>
```

File: website/blog/2021-05-31-tungsten.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
```
Comment (tgodzik) [inline website/blog/2021-05-31-tungsten.md:35]:
```suggestion
For full details: https://github.com/scalameta/metals/milestone/39?closed=1
```

File: website/blog/2021-05-31-tungsten.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+give Metals a try!
+
+- Add support for Scala 2.12.14
+
+## Contributors
+
+Big thanks to everybody who contributed to this release or reported an issue!
+
+```
+$ git shortlog -sn --no-merges v0.10.3..v0.10.4
```
Comment (tgodzik) [inline website/blog/2021-05-31-tungsten.md:55]: I usually process it a bit, but it's not 100% needed I guess.

```suggestion
$ git shortlog -sn --no-merges v0.10.3..v0.10.4
	Tomasz Godzik
	Vadim Chelyshov
	Luigi Frunzio
	Chris Kipp
```

**Review by tgodzik — CHANGES_REQUESTED** [review]
Thanks for making the release notes! Just some minor comments, otherwise we can start the release.

**Review by Giggiux — APPROVED** [review]
LGTM!

**Review by tgodzik — APPROVED** [review]
LGTM!

**Pull request merged**